pandapower/converter/cim/cim2pp/converter_classes/transformers/tapController.py

```diff
@@ -24,44 +24,51 @@ def create_tap_controller_for_power_transformers(self):
-                self._create_characteristic_object(net=self.cimConverter.net, trafo_type='trafo', trafo_id=[trafo_id],
-                                                   characteristic_df=characteristic_df)
-            self.logger.info(f"Finished creating tap dependent impedance characteristic objects for 2w-trafos in "
-                             f"{time.time() - time_start}.")
+            # moved out of converter
+            # time_start = time.time()
+            # self.logger.info("Creating the tap dependent characteristic objects for 2w-trafos.")
+            # # create the characteristic objects for transformers
+            # characteristic_df_temp = \
+            #     self.cimConverter.net['trafo_characteristic_table'][
+            #         ['id_characteristic', 'step', 'voltage_ratio', 'angle_deg', 'vk_percent', 'vkr_percent']]
+            # for trafo_id, trafo_row in self.cimConverter.net.trafo.dropna(
+            #         subset=['id_characteristic_table']).iterrows():
+            #     characteristic_df = characteristic_df_temp.loc[
+            #         characteristic_df_temp['id_characteristic'] == trafo_row['id_characteristic_table']]
+            #     self._create_trafo_characteristic_object(net=self.cimConverter.net, trafo_type='trafo',
+            #                                              trafo_id=[trafo_id], characteristic_df=characteristic_df)
+            # self.logger.info(f"Finished creating tap dependent characteristic objects for 2w-trafos in "
+            #                  f"{time.time() - time_start}.")
         if self.cimConverter.power_trafo3w.index.size > 0:
             # create transformer tap controller
             self._create_tap_controller(self.cimConverter.power_trafo3w, 'trafo3w')
-            time_start = time.time()
-            self.logger.info("Creating the tap dependent impedance characteristic objects for 3w-trafos.")
-            # create the characteristic objects for transformers
-            characteristic_df_temp = \
-                self.cimConverter.net['characteristic_temp'][
-                    ['id_characteristic', 'step', 'vkr_hv_percent', 'vkr_mv_percent',
-                     'vkr_lv_percent', 'vk_hv_percent', 'vk_mv_percent', 'vk_lv_percent']]
-            for trafo_id, trafo_row in self.cimConverter.net.trafo3w.dropna(subset=['id_characteristic']).iterrows():
-                characteristic_df = characteristic_df_temp.loc[
-                    characteristic_df_temp['id_characteristic'] == trafo_row['id_characteristic']]
-                self._create_characteristic_object(net=self.cimConverter.net, trafo_type='trafo3w', trafo_id=[trafo_id],
-                                                   characteristic_df=characteristic_df)
-            self.logger.info(f"Finished creating tap dependent impedance characteristic objects for 3w-trafos in "
-                             f"{time.time() - time_start}.")
+            # moved out of converter
```

Comment:
is this code still needed?

Reply:
no, you're right. I removed the commented code and committed my changes, thanks